cmd/aws-iam-authenticator/server.go

```diff
@@ -30,8 +30,14 @@ import (
 	"github.com/spf13/viper"
 )
 
-// DefaultPort is the default localhost port (chosen randomly).
-const DefaultPort = 21362
+
+const (
+	// DefaultPort is the default localhost port (chosen randomly).
+	DefaultPort                         = 21362
+	// Default Ec2 TPS Variables
```

Comment:
Nit: Add a newline before the comment so that `gofmt` will intent this a bit nicer.